src/main/java/GeneralService/OnlineServicesCommunicationService.java

```diff
@@ -0,0 +1,51 @@
+package GeneralService;
+
+import onlineservices.services.WindowControl.WindowControlService;
+import org.eclipse.paho.client.mqttv3.*;
```

Comment:
remove wildcard import with specific imports, i don't want everything even if 90% might not be used, only import what you use